evaluation/run_eval.py

```diff
@@ -60,6 +60,12 @@ def parse_args():
         default=None,
         help="The configuration name of the dataset to use (via the datasets library).",
     )
+    parser.add_argument(
+        "--template_config_name",
+        type=str,
+        default=None,
+        help="The name of the dataset_config_name of the template we want to use, example: use XNLI En prompts for XNLI Fr",
```

Comment:
Is using english prompts on the french XNLI something good? Like I understand it's some sort of measure multilinguality, but I would have expected to have a bunch of frnech prompts for the XNLI fr ...

Reply:
I thought using english prompts for XNLI is indeed what we wanted to accomplish?

Anyway, evaluating XNLI with prompts and dataset in the same language only requires the Promptsource part be updated. So `--template_config_name` adds more flexibility while not needing to change too much of the code to be able to eval on multiple multilingual settings.